src/initramfs-tools/Makefile

```diff
@@ -5,12 +5,12 @@ SHELL = /bin/bash
 MAIN_TARGET = initramfs-tools_$(INITRAMFS_TOOLS_VERSION)_all.deb
 DERIVED_TARGETS = initramfs-tools-core_$(INITRAMFS_TOOLS_VERSION)_all.deb
 
-INITRAMFS_TOOLS_REVISION = 18fc98e1b63b012f9bcf06ae3f5477872a5880c0
+INITRAMFS_TOOLS_REVISION = 40e544e
```

Comment:
>40e544e [](start = 27, length = 7)

Use a long hash? It may be invalid if the upstream grows.

Reply:
done